src/components/Utils/filters.js

```diff
@@ -75,15 +75,24 @@ export const updateCountsInInitialTabsOptions = (
           if (flattenProcessedTabsOptions[`${field}`]
             && flattenProcessedTabsOptions[`${field}`].length > 0
             && updatedTabsOptions[`${actualFieldName}`].histogram) {
-            updatedTabsOptions[`${actualFieldName}`].histogram[0].count = flattenProcessedTabsOptions[`${field}`][0].count;
+            const currentFlattenProcessedTabsOptions = flattenProcessedTabsOptions[`${field}`][0];
+
+            // if empty range histogram should be removed so filter is not shown
+            if (currentFlattenProcessedTabsOptions.key[0] === null
+              && currentFlattenProcessedTabsOptions.key[1] === null
+              && currentFlattenProcessedTabsOptions.count === 0) {
```

Comment:
should be enough if we just check for `count === 0` ?

Reply:
I was being extra cautious, will run some tests and see if anything ever pops up with a count that does not have a range

Reply:
updated to only check count